lstchain/scripts/lstchain_dl1_to_dl2.py

```diff
@@ -43,68 +44,63 @@
 from lstchain.reco import dl1_to_dl2
 from lstchain.reco.utils import filter_events, impute_pointing, add_delta_t_key
 
-parser = argparse.ArgumentParser(description="DL1 to DL2")
+parser = argparse.ArgumentParser(description=
```

Comment:
I'd suggest to actually use a multiline string here:

```
description = '''Run the DL1 to DL2 step.
...
""")

This will look much better in the terminal.

Reply:
Thanks. I also simplified the description, I forgot that `--help` was already giving the example usage.